feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts

```diff
@@ -60,12 +61,38 @@ export class ConfiguratorTabBarComponent {
       )
     );
 
+  /**
+   * @deprecated Use getPageType$ and isOverviewPage(ConfiguratorRouter.PageType)
+   * instead
+   */
   isOverviewPage$: Observable<boolean> = this.routerData$.pipe(
     map(
       (routerData) =>
         routerData.pageType === ConfiguratorRouter.PageType.OVERVIEW
     )
   );
+  /**
+   * Retrieves current page type
+   * @returns - page type
+   */
+  pageType$: Observable<ConfiguratorRouter.PageType> = this.routerData$.pipe(
+    map((routerData) => this.determinePageFromRouterData(routerData))
+  );
+
+  protected determinePageFromRouterData(
+    routerData: ConfiguratorRouter.Data
+  ): ConfiguratorRouter.PageType {
+    return routerData.pageType ?? ConfiguratorRouter.PageType.CONFIGURATION;
+  }
+
+  /**
+   * Checks whether the current page is the overview page.
```

Comment:
The same here:
Please add an empty line between an explanation and a  return value.